### DISCUSSION TITLE
When kubelet is restarted, the endpoints of the service are lost temporarily.

### DISCUSSION THREAD

**Discussion by zhaizeyu** [kedacore/http-add-on#1124] [Q&A]
## report
My k8s version is 1.22.1,
 service type is Cluster. 
When i restart kubelet and use this cmd kubectl get endpoints fkft7-nslb-north-svc -nns000000000000000000001，i find the endpoints of the service are lost temporarily. and log of kube-controller-manager show me like this
![image](https://github.com/user-attachments/assets/64a12a03-3e7c-4b0e-9d75-5aaef607bdcc)

Are other conditions triggering this bug as well? I wonder if this problem has been fixed.

**Comment by linkvt** [comment]
Closing as outdated — this predates several releases and either got resolved since or never reached a conclusion. See current docs (https://keda.sh/http-add-on/0.15/) or feel free to open a new discussion if still relevant.
